### ISSUE TITLE
Suggestion - More ideas for EMS

### ISSUE THREAD

**Issue by coalaura** [InZidiuZ/op-framework-issue-tracker#3514]
More ideas for EMS

**Eating Expired Items:**

- Players can consume expired food or drinks to reduce hunger or thirst.
- Consuming expired items can lead to illnesses with gradual effects.

**Illness Effects:**

- **Diarrhea:** Causes abnormal thirst level reduction.
- **Fever:** Results in health loss over time.
- **Headaches:** Increases stress level slightly (e.g., by 0.1).

**Role-Playing with EMS:**

- Players experiencing abnormal thirst, stress, or health issues can RP with EMS to get treatment.
- Vomit emote: Players can vomit after consuming expired items, reducing hunger level.

**Medical Interactions:**

- **Checkups:** EMS can perform checkups using a thermometer for fever and other medical conditions.
- **Blurred Vision:** A rare effect (1 in 100) where players experience blurred vision, leading to potential RP scenarios for eye surgery.

**Vision Correction:**

- Introduce an item like glasses with different power levels to correct vision.
- If the glasses are not in the player's possession, they experience blurred vision, prompting them to seek RP with EMS for corrective measures.

_Originally posted by @Pixel-Nomad in https://github.com/InZidiuZ/op-framework-issue-tracker/issues/3512#issuecomment-2286878255_

**Comment by PurpleHzea** [comment]
Eating expired/rotten food should 100% cause life threatening illnesses and that could be a good addition for the server. But the forced blurred vision is a bit wack IMHO. Maybe instead of just checking in with nancy and EMS checking people in with her, doctors would actually have to work on the patients that come into the hospital and It takes longer than just a single minute to treat the patients. But, that would require only one main hospital when EMS are on duty and would cause a lot of people to complain 

TLDR: Remove nancy and have EMS/doctors actually treat patients rather than send them to nancy. Lockup other hospitals besides the main two [Mt. Z & Saint. F] while there are doctors on duty

**Comment by PurpleHzea** [comment]
Seems like the main issue with EMS rn is the lack of Roleplay. Go to scene, transport patient, check in with nancy, leave. People get too caught up for slowburn RP which is exactly what EMS RP is IMO. So I feel like trying a different method wouldn't hurt

**Comment by Pixel-Nomad** [comment]
i want to make a change a little bit replace glasses with lens item since we can't force anyone to wear glasses

**Comment by SammySandbags** [comment]
More ideas for EMS

Eating Expired Items:

Players can consume expired food or drinks to reduce hunger or thirst.
Consuming expired items can lead to illnesses with gradual effects.
Illness Effects:

Diarrhea: Causes abnormal thirst level reduction.                     **Treated with a new Item from the EMS Pharmacy [bismuth subsalicylate (Pepto-Bismol)]** 
Fever/Food Poising: Results in health loss over time.                **Treated with a dose of Antibiotics from the EMS Pharmacy** 
Headaches: Increases stress level slightly (e.g., by 0.1).             **Treated with a dose of painkillers from the EMS Pharmacy**
Role-Playing with EMS:

Can we Change the Painkillers to Read as or add  TYLENOL as a separate Pain Medication. The reason for this would be to allow Paramedics to issue Tylenol to Patients and the Painkiller as a prescribed medication only issued by doctors. 

The Blurry Vision could maybe have 2 routes for correction a Simple one of EYE Drops Issued again from the EMS Pharmacy Another new item. The second one would be for a doctors appointment for those that want a more in depth treatment. 


Checkups: EMS can perform checkups using a thermometer for fever and other medical conditions.   Great Idea would love to have this as a thing we can use to actually get peoples temperatures. 
/checktemp    then a few seconds wait and a return of Temperature of ****

Gives a random Value if they have the Below Problems:

No Injuries would return a value between 97 - 102 F
If they have a food related injury would be in the range of 102.4 to 105.8 F  (Fever)
If they have a blood loss related Injury would be between the ranges of  90 - 95 (broken limbs Ect the normal stuff we have now).

**Comment by SammySandbags** [comment]
> > i want to make a change a little bit replace glasses with lens item since we can't force anyone to wear glasses
> 
> I would not add effects that force a certain roleplay path. Things should be optional with nancy as the backup in case nobody is around

Exactly

**Comment by battlea** [comment]
> Seems like the main issue with EMS rn is the lack of Roleplay. Go to scene, transport patient, check in with nancy, leave. People get too caught up for slowburn RP which is exactly what EMS RP is IMO. So I feel like trying a different method wouldn't hurt

People can make up their own injuries and request a doctor if they want more medical roleplay. Paramedics are also trained to handle basic injuries and can give someone a cast if no surgery is needed

**Comment by spitslave** [comment]
Love everything about this IDEA

**Comment by spitslave** [comment]
Best idea
